gradle/libs.versions.toml

```diff
@@ -4,19 +4,22 @@ ideProbe = "0.36.1"
 powerMock = "2.0.9"
 
 [libraries]
+# plugin dependency notation
```

Comment:
If this stays for good (and is NOT refactored away over #1004), then:
1. the versions shared with the `[plugins]` sections should be extracted to `[versions]` section to avoid discrepancies
2. extra care should be taken to retain `@pin`s on certain plugins, as done in `[plugins]` currently (so that they're NOT auto-updated by `gradle versionCatalogUpdate`

Pls either resolve on this PR, or update #1004

Reply:
Moved to #1004